lib/Db/Helper.php

```diff
@@ -101,10 +101,10 @@ public static function queryIgnoreError(Connection $db, string $sql, array $excl
     public static function quoteInto(Connection $db, string $text, mixed $value, int|string|Type|null $type = null, ?int $count = null): array|string
     {
         if ($count === null) {
-            return str_replace('?', $db->quote($value, $type), $text);
+            return str_replace('?', $db->quote($value), $text);
```

Comment:
Since with Doctrine 4, the quote only accepts a string, we need to change the mixed value to string value and we also need to remove the type.
Problem is, that this is a BC Break.
We cannot use Version 3 and 4 in parallel unless we do some magic checking. Will affect all the tests etc.